Funcky.Async/Extensions/AsyncEnumerableExtensions/Interleave.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections.Immutable;
+
+namespace Funcky.Async.Extensions
+{
+    public static partial class AsyncEnumerableExtensions
+    {
+        /// <summary>
+        /// Interleaves the elements of multiple sequences by consuming the heads of each subsequence in the same order as the given subsequences. This repeats until all the sequences are completley consumed.
+        /// </summary>
+        /// <typeparam name="TSource">The type of the elements in the source sequences.</typeparam>
+        /// <param name="sequence">first sequence.</param>
+        /// <param name="otherSequences">other sequences.</param>
+        /// <returns>one sequences with all the elements interleaved.</returns>
+        [Pure]
+        public static IAsyncEnumerable<TSource> Interleave<TSource>(this IAsyncEnumerable<TSource> sequence, params IAsyncEnumerable<TSource>[] otherSequences)
+            => ImmutableList.Create(sequence).AddRange(otherSequences).Interleave();
+
+        /// <summary>
+        /// Interleaves the elements of a sequence of sequences by consuming the heads of each subsequence in the same order as the given subsequences. This repeats until all the sequences are completley consumed.
+        /// </summary>
+        /// <typeparam name="TSource">The type of the elements in the source sequences.</typeparam>
+        /// <param name="source">the source sequences.</param>
+        /// <returns>one sequences with all the elements interleaved.</returns>
+        [Pure]
+        public static async IAsyncEnumerable<TSource> Interleave<TSource>(this IEnumerable<IAsyncEnumerable<TSource>> source)
+        {
+            var enumerators = GetInterleaveEnumerators(source);
+
+            try
+            {
+                await foreach (var element in InterleaveEnumeratorAsync(enumerators))
```

Comment:
Missing `.ConfigureAwait`

Reply:
Fixed in 835f1107a4109d986872ea7943e60e9930dfb624